app/src/lang/translations/fr-CA.yaml

```diff
@@ -1231,6 +1231,7 @@ translation: Traduction
 translation_placeholder: Entrez une nouvelle traduction...
 value: Valeur
 view_project: Voir le projet
+weeks: { }
```

Comment:
@azrikahar do you think this is OK or is a bug?

Reply:
Should be fine. Just results in an empty object, which in turn means vue-i18n will default to the english names for the weeks.